backend/src/services/event_service.py

```
@@ -311,6 +311,185 @@ def get_stats(self, team_id: int) -> dict:
+            - needs_pto_count: Events where timeoff_required and status != 'approved'
+            - needs_travel_count: Events where travel_required and status != 'booked'
+        """
+        today = date.today()
+        end_date = today + timedelta(days=30)
+
+        # Common filters: future events, non-deleted, non-deadline
+        common_filters = [
+            Event.team_id == team_id,
+            Event.event_date >= today,
+            Event.status.in_(["future", "confirmed"]),
+            Event.is_deadline == False,
```

Ruff (0.15.0)

[error] Avoid equality comparisons to `False`; use `not Event.is_deadline:` for false checks

Replace with `not Event.is_deadline`

(E712)